### ISSUE TITLE
ERROR: Misconfigured. No 'class' in configuration for ml:adhoc-query

### ISSUE THREAD

**Issue by grtjn** [ndw/xmlcalabash1#261]
I was looking into https://github.com/xquery/xquerydoc, and tried running its test suite using xmlcalabash-1.1.16-97, since the provided calabash seems outdated and misconfigured. Unfortunately, I am getting this message:

    ERROR: Misconfigured. No 'class' in configuration for ml:adhoc-query

I installed xmlcalabash using the provided installer under /Applications/xmlcalabash-1.1.16-97/, and made a symlink from /usr/local/bin/calabash to the shell script in that folder.

Is there some obvious mistake I can easily fix, or is this an internal issue with xmlcalabash?

**Comment by ndw** [comment]
Is it possible that you have an old version of the XCC plugins? https://github.com/ndw/xmlcalabash1-xcc

**Comment by grtjn** [comment]
Well, I downloaded, and unzipped the latest, but wasn't sure how to proceed next. I'm using calabash command-line script. How can I tell it I have an extra jar that I'd like to load?

**Comment by grtjn** [comment]
Ok, I dropped `xmlcalabash1-xcc-1.1.3.jar` next to `xmlcalabash-1.1.16-97.jar` in `/Applications/xmlcalabash-1.1.16-97/` (yes, running on my mac), and added latest ml xcc jar (from mlcp) in the `lib` subfolder.

I then changed the calabash script java call into:

    java -Xmx1024m -cp /Applications/xmlcalabash-1.1.16-97/lib/*:/Applications/xmlcalabash-1.1.16-97/* com.xmlcalabash.drivers.Main "$@"

And now I am getting past the earlier message.

Is this the only way, or is there a better approach? Maybe useful to use `../lib/*:../*` in the calabash script by default, saves an extra step. Or allow feeding a CLASSPATH from outside..
